client/dashboard/sites/settings-defensive-mode/index.tsx

```diff
@@ -210,7 +210,22 @@ export default function DefensiveModeSettings( { siteSlug }: { siteSlug: string
 	};
 
 	return (
-		<PageLayout size="small" header={ <SettingsPageHeader title={ __( 'Defensive mode' ) } /> }>
+		<PageLayout
+			size="small"
+			header={
+				<SettingsPageHeader
+					title={ __( 'Defensive mode' ) }
+					description={ createInterpolateElement(
+						__(
+							'Extra protection against spam bots and attacks. Visitors will see a quick loading page while we run additional security checks. <learnMoreLink />'
```

Comment:
Seems like we need to decide whether to use this "style" or `... security checks. <a>Learn more</a>` (i.e. whether to split the strings or not).

We can reconcile later.

Reply:
Yeah, I saw different approaches, but I expect that to be resolved when we hook it up.